src/query/sql/src/executor/physical_plan.rs

```diff
@@ -777,6 +777,7 @@ pub struct MergeInto {
     // the first option stands for the condition
     // the second option stands for update/delete
     pub matched: MatchExpr,
+    pub filed_index_of_input_schema: HashMap<FieldIndex, usize>,
```

Comment:
better to add some comments to explain the usage of `filed_index_of_input_schema`